azure-client-authentication/src/main/java/com/microsoft/azure/credentials/AzureCliCredentials.java

```diff
@@ -118,6 +119,13 @@ public AzureEnvironment environment() {
         return subscriptions.get(defaultSubscriptionId()).environment();
     }
 
+    /**
+     * @return the subscriptions the current account in Azure CLI has access to
+     */
+    public Map<String, AzureCliSubscription> subscriptions() {
```

Comment:
I think you should also make `class AzureCliSubscription` public, it's not 'visible' from outside packages at current.